src/components/organisms/Footer.js

```diff
@@ -1,9 +1,88 @@
-import React from 'react'
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
The project is built on tailwind CSS strictly. Please avoid using bootstrap at all in this codebase.